pkg/kubectl/cmd/apply_diff_last_applied.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/util/diff"
+)
+
+var (
+	applyDiffLastAppliedLong = templates.LongDesc(`
+		Opens up a 2-way diff in the default diff viewer. This should follow the same semantics as git diff.
+		It should accept either a flag --diff-viewer=meld or check the environment variable KUBECTL_EXTERNAL_DIFF=meld.
```

Comment:
the problem is [here](https://github.com/shiywang/kubernetes/blob/2b178ad608bd443efeaf3e9ea7c700e72ab68de1/pkg/kubectl/cmd/templates/normalizers.go#L73), this markdown function and `ASCIIRenderer` @fabianofranz do you have any ideas on this ? 
and more accurately `KUBE_EDITOR` also have problem, it render `KUBE_EDITOR` to `KUBE_ EDITOR` if you watch carefully